### DISCUSSION TITLE
Is it time to yank out import maps?

### DISCUSSION THREAD

**Discussion by caspervonb** [denoland/deno#8762] [General]
While the proposal itself is still being kept alive, Import maps aren't seeing alot (or any rather) traction with vendors.

Is it time to yank the plug on them? What kind of usage is it seeing in the wild, considering it's an --unstable feature?

**Comment by caspervonb** [comment]
Oh actually @lucacasonato dug this up from Chrome https://groups.google.com/a/chromium.org/g/blink-dev/c/rVX_dJAJ-eI/m/17r-6-eiAgAJ so might have jumped the gun here.

**Reply by kitsonk** [reply]
Yup, it is intent to ship on Chromium now.  Finally getting there.  Plus we don't have any alternative, and there are a lot of valid use cases for remapping imports.
